app/src/main/java/com/aspirebudgetingmobile/aspirebudgeting/activities/Login.java

```diff
@@ -0,0 +1,82 @@
+package com.aspirebudgetingmobile.aspirebudgeting.activities;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.widget.RelativeLayout;
+import android.widget.Toast;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.core.content.ContextCompat;
+
+import com.aspirebudgetingmobile.aspirebudgeting.R;
+import com.aspirebudgetingmobile.aspirebudgeting.utils.ObjectFactory;
+import com.google.android.gms.auth.api.signin.GoogleSignIn;
+import com.google.android.material.card.MaterialCardView;
+
+public class Login extends AppCompatActivity {
+
+    private static final String TAG = "LOGIN_ACTIVITY";
+
+    RelativeLayout googleRelativeLayout;
+    MaterialCardView googleLoginCard_login;
+
+    ObjectFactory objectFactory = ObjectFactory.getInstance();
+    int GOOGLE_SIGN_IN = 0;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_login);
+
+        if (getWindow() != null) {
+            // SETTING THE NAVIGATION BAR WHITE
+            getWindow().setNavigationBarColor(ContextCompat.getColor(Login.this, android.R.color.white));
+        }
+
+        // CREATE AN OBJECT OF USER_MANAGER AND INITIALIZE GOOGLE SIGN IN
+        objectFactory.getUserManager().initializeGoogleSignIn(Login.this);
```

Comment:
Store an instance of UserManager in this class instead of calling getUserManager() every time you need to use it.

Reply:
done